app/lib/account/consent_backend.dart

```diff
@@ -411,7 +422,10 @@ class _PublisherMemberAction extends ConsentAction {
   Future<void> onAccept(Consent consent) async {
     final member = consent.userId != null
         ? await accountBackend.lookupUserById(consent.userId)
-        : await accountBackend.lookupOrCreateUserByEmail(consent.email);
+        : await accountBackend.lookupUserByEmail(consent.email);
+    if (member == null) {
+      throw AuthenticationException.userNotFound();
```

Comment:
Maybe add a command saying: this should never happen because userId of the consent entity will be set when it is loaded.